lib/bundles/inspec-compliance/cli.rb

```diff
@@ -30,26 +30,22 @@ def login(server) # rubocop:disable Metrics/AbcSize
       url = options['server'] + options['apipath']
       if !options['user'].nil? && !options['password'].nil?
         # username / password
-        success, msg = login_legacy(url, options['user'], options['password'], options['insecure'])
+        msg = login_legacy(url, options['user'], options['password'], options['insecure'])
       elsif !options['user'].nil? && !options['token'].nil?
         # access token
-        success, msg = store_access_token(url, options['user'], options['token'], options['insecure'])
+        msg = store_access_token(url, options['user'], options['token'], options['insecure'])
```

Comment:
oh, oops, sorry. i took that out cause of a lint complaint, I should have just left it in with a leading underscore....late night changes :)  I'll switch it up